app/services/SicAndComplianceService.scala

```diff
@@ -121,4 +121,6 @@ class SicAndComplianceService @Inject()(val s4lService: S4LService,
   }
 }
 
-case class LandAndPropertyAnswer(answer: Boolean)
\ No newline at end of file
+case class LandAndPropertyAnswer(answer: Boolean)
+
+case class OtherBusinessInvolvementAnswer(answer: Boolean)
```

Comment:
updateModel def should be updated to handle this model to actually be able to store it, similar to how it does with LandAndPropertyAnswer